reconcile/openshift_resources_base.py

```diff
@@ -242,7 +242,9 @@ def process_jinja2_template(body, vars=None, env=None):
     return r
 
 
-def process_extracurlyjinja2_template(body, vars={}):
+def process_extracurlyjinja2_template(body, vars=None):
+    if not vars:
```

Comment:
This may be a nit for these specific use cases, but I generally err on the side of checking for `None` directly with `if vars is None` because we care about whether it was set or not more than what it is a truthy value or not?

Reply:
I believe you are right it should be best to check if the value is None. I'll make the change across all occurences

Reply:
fixed!